app/src/main/java/com/example/habithelper/fragments/ListFragment.java

```diff
@@ -2,17 +2,37 @@
 
 import android.os.Bundle;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
 import com.example.habithelper.R;
+import com.example.habithelper.activities.MainActivity;
+import com.example.habithelper.adapters.HabitAdapter;
+import com.example.habithelper.models.Habit;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
 
 
 public class ListFragment extends Fragment {
```

Comment:
ListFragment as a name is not descriptive. We would prefer to specify list of what